.babelrc

```diff
@@ -1,3 +1,4 @@
 {
-  "presets": ["@babel/preset-env", "@babel/preset-react"]
+  "presets": ["@babel/preset-env", "@babel/preset-react"],
+  "plugins": ["css-modules-transform"]
 }
```

Comment:
I don't use a babel rc so maybe this is not needed. Please consider deleting this file.